src/streamer/stream.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { StatusResult } from '@salesforce/core';
+import { AnyJson, JsonMap } from '@salesforce/ts-types';
+
+export default interface Stream {
```

Comment:
```suggestion
/**
* Common interface used to monitor the progress of something in the DevOps Center.
**/
export default interface DoceMonitor {
```

I think "Stream" is too dependent on the implementation.  It does not make as much sense if we implement a polling version of this...right?